spec/README.md

```diff
@@ -0,0 +1,22 @@
+Jakarta Concurrency Specification
+============================
+
+This project generates the Jakarta Concurrency 1.1 Specification.
```

Comment:
Just to make sure... This project does not do that, right? Like in top dir it's about `Jakarta™ Connectors`, and this README belongs to [eclipse-ee4j/concurrency-api](https://github.com/eclipse-ee4j/concurrency-api).